images/user-agent/Dockerfile

```diff
@@ -1,3 +1,7 @@
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
For some reason locally the license text check fails for me on this file now. The CI doesn't seem to have a problem, though.